src/test/java/bio/overture/ego/controller/TokensOnPermissionsChangeTest.java

```diff
@@ -499,13 +499,13 @@ private String userPermissionTestSetup(
    * @param user The user that will belong to the group which is having the permissions mutated.
    * @param group The group to which the group permissions are assigned.
    * @param policy The policy that the permission is relevant to.
-   * @param initalAccess The initial access level (MASK) for the group on the policy.
+   * @param initialAccess The initial access level (MASK) for the group on the policy.
    * @param tokenScopeSuffix The scope suffix that the token will be created with for the user.
    * @return The access token
```

Comment:
`The access token`  to `The API Key`

Reply:
Does it make sense to rename the other `access token` references to `apiKey` in the other test scenarios? i.e. in descriptions like
`/**
   * Scenario: User is part of a group that has a WRITE permission. User has a token using this
   * scope. The group then has the permission deleted. Behavior: Token should be revoked.
   */`